src/TTextEdit.cpp

```diff
@@ -1354,6 +1372,53 @@ void TTextEdit::copySelectionToClipboardHTML()
     return;
 }
 
+void TTextEdit::searchSelectionOnline()
+{
```

Comment:
Mind sprinkling a few comments on this function explaining the "whys" at different stages?

Reply:
I could move most of it to a `QString getSelectedText()` of sort while I am at it, since it's used in the `copySelectionToClipboard()` as well. Anything wrong with that?

Reply:
That sounds like a good abstraction to do, I agree.